themes/bootstrap5/js/get_this_dropdown.js

```diff
@@ -0,0 +1,121 @@
+let KEY_CODE = {
+  DOWN_ARROW: 40,
+  UP_ARROW: 38,
+};
+$(function pageLoad() {
+  let modal = document.querySelector('#modal');
+
+  /**
+   * Whether the dropdown is open
+   * @returns {boolean} Whether the dropdown is open
+   */
+  function isDropdownOpen() {
+    return modal.querySelector(".get-this-dropdown > ul > li > div").classList.contains("active");
+  }
+
+  /**
+   * Whether the link for the current holding is the element being clicked
+   * @param e Event triggered by the user
```

Comment:
Take a look at the file diffs in the PR and you'll see a few missing type errors from GitHub Actions.